solvation_analysis/tests/test_plotting.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pytest
+
+from solvation_analysis.plotting import (
+    plot_network_size_histogram,
+    plot_shell_size_histogram,
+    compare_pairing,
+    compare_coordination_numbers,
+    compare_residence_times,
+    compare_speciation,
+    format_graph,
+)
+
+
+def test_plot_network_size_histogram(networking):
+    fig = plot_network_size_histogram(networking)
+    fig.show()
+    assert True
```

Comment:
I'd always assert something meaningful, e.g., that this is the correct instance, or you can dig into the figure object and check that it has the right labels, or that the data is what you expect it to be. But do a tiny bit more than flushing code paths.